requirements.txt

```diff
@@ -1,8 +1,9 @@
-django==1.8
+django<1.9
```

Comment:
It is a really good idea to only specify exact version numbers here. That way, you won't run into issues where differing versions in prod and dev create subtle and hard-to-discover bugs due to differences in versions. You can use pip freeze to see the current version numbers installed in an environment.